code-challenges.test.js

```diff
@@ -65,5 +121,45 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe('vowels', () => {
+    it("takes in a string and logs the index of the first vowel")
+    const vowelTester1 = "learn"
+    const vowelTester2 = "academy"
+    const vowelTester3 = "challenges"
+    expect(vowelTester1).toEqual(vowelTester1)
+    expect(vowelTester2).toEqual(vowelTester2)
+    expect(vowelTester3).toEqual(vowelTester3)
+
+})
+
+//Pseudocode
+// declare variable 'vowels'
+// make a  3 parameters 'vowelTester1, vowelTester2, vowelTester3
+// use .filter to only return index []
+
+const vowels = (vowelTester1, vowelTester2, vowelTester3) => {
+    return vowelTester1, vowelTester2, vowelTester3 => {
+        value.
+    }
+
+}
+
+
 
 // b) Create the function that makes the test pass.
+
+//---------------------- NOTES-------------------------
+// - had alot of problems with the problems with the challenges
+
+     //All the problems read outputed below and unsure what that means
+// > 3 | const { default: test } = require("node:test")
+// |                           ^
+// 4 | const { isTypedArray } = require("util/types")
+// 5 | const { describe, array } = require("yargs")
+//---------------------------------------------------------------------
+//                  PROBLEM 2
+// 1.) I'm unsure if I'm correct but there's a curly bracket that give me an error and everytime I try to adjust problems on different parts of the code happen
```

Comment:
Line 97 and 99 might need to swap.  In VS code, uou can select a curly brace and your text editor will highlight it's pair that it belongs to.  This can be helpful in this scenario.